RELEASING.md

```diff
@@ -16,11 +16,14 @@ $ # get the PR reviewed and merged
 ## Release
 
 ```
+$ # Ensure you are in a clone of the main repository, NOT your fork
```

Comment:
Being pedantic but I think you're just talking about the `origin` being not your fork. Is this just for pushing the tags?

Reply:
Ah, I was close enough 😆